libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -60,7 +63,7 @@ class AC_PosControl
     void input_pos_xyz(const Vector3p& pos, float pos_offset_z, float pos_offset_z_buffer);
 
     /// pos_offset_z_scaler - calculates a multiplier used to reduce the horizontal velocity to allow the z position controller to stay within the provided buffer range
-    float pos_offset_z_scaler(float pos_offset_z, float pos_offset_z_buffer);
+    float pos_offset_z_scaler(float pos_z, float pos_offset_z_buffer);
```

Comment:
We should change the argument name in the method's implementation as well.  Put another way, the argument names don't match between the .h and .cpp files...

Reply:
by the way, I think this method can be made const but this is not a requirement for this PR in any case.